packages/block-library/src/paragraph/edit.js

```diff
@@ -155,6 +159,23 @@ class ParagraphBlock extends Component {
 							setAttributes( { align: nextAlign } );
 						} }
 					/>
+					{ isRTL && (
+						<Toolbar
+							controls={ [
+								{
+									icon: 'editor-ltr',
+									title: _x( 'Left to right', 'editor button' ),
```

Comment:
In case this raises eyebrows, the hope / intention is to inherit the existing string:

https://github.com/WordPress/WordPress/blob/da7a80d67fea29c2badfc538bfc01c8a585f0cbe/wp-includes/class-wp-editor.php#L1128